onadata/settings/base.py

```diff
@@ -6,11 +6,13 @@
 from datetime import timedelta
 from urllib.parse import quote_plus
 
-import dj_database_url
 from django.core.exceptions import SuspiciousOperation
+import environ
```

Comment:
FYI: https://chat.kobotoolbox.org/#narrow/stream/4-Kobo-Dev/topic/Python.20coding.20style/near/3236

So, FYI, following our coding style rules.
`import ...` should come before `from ...`